addons/udes_stock/models/stock_picking.py

```diff
@@ -2724,20 +2724,6 @@ def mark_empty_pickings(self):
             _logger.info(_("Flagging empty pickings for clean up: %r") % empty_picks.ids)
             empty_picks.write({"u_mark": False})
 
-    def _unassign_picker(self, **kwargs):
-        """
-        Method to clear the user from pickings.
-        Designed to be overwritten by other more specific modules.
-        """
-        return
-
-    def _assign_picker(self, **kwargs):
```

Comment:
I think there are some places in udes_workflows module that call _assign_picker and _unassign_picker.
IF removed it will raise error on udes_workflows , if module udes_kafka_events is not installed as the method is initially override in that module too.